lib/scan/admin/engine.py

```diff
@@ -240,13 +239,12 @@ def start(target, users, passwds, ports, timeout_sec, thread_number, num, total,
                 break
         thread_write = int(open(thread_tmp_filename).read().rsplit()[0])
         if thread_write is 1:
-            info(messages(language, "directory_file_404").format(
-                target, "default_port"))
             if verbose_level is not 0:
-                data = json.dumps(
+                info(messages(language, "directory_file_404").format(
+                    target, "default_port"), log_in_file, "a",
                     {'HOST': target_to_host(target), 'USERNAME': '', 'PASSWORD': '', 'PORT': '', 'TYPE': 'admin_scan',
-                     'DESCRIPTION': messages(language, "no_open_ports"), 'TIME': now(), 'CATEGORY': "scan", 'SCAN_ID': scan_id,
-                     'SCAN_CMD': scan_cmd})
+                     'DESCRIPTION': messages(language, "direcroty_file_404").format(target, "default_port"), 'TIME': now(), 'CATEGORY': "scan", 'SCAN_ID': scan_id,
```

Comment:
@pradeepjairamani, you have replaced "no_open_ports" with "direcroty_file_404". Is this done intentionally? If so, you have mistyped the spelling for directory as direcroty. Please correct this!!

Reply:
This was done intentionally and thanks for letting me know about the spelling mistake